cli.js

```diff
@@ -27,6 +28,25 @@ If file/glob is omitted, './package.json' file will be processed.
   )
 }
 
+function parseCliArguments() {
+  const { values: options, positionals: patterns } = parseArgs({
+    options: {
+      check: { type: 'boolean', short: 'c', default: false },
+      quiet: { type: 'boolean', short: 'q', default: false },
+      version: { type: 'boolean', short: 'v', default: false },
+      help: { type: 'boolean', short: 'h', default: false },
```

Comment:
[Docs says `default` only supported in Node.js v18](https://nodejs.org/api/util.html#utilparseargsconfig), since our flags all default to `false`, so it really matters.

Reply:
It says that, but running the program in node js v16.19.0, the default field works

Reply:
Click `history` under title.

Reply:
![image](https://user-images.githubusercontent.com/172584/216250502-cce98d7f-c5e0-4a21-b5e0-7c9724772f5c.png)

Reply:
I already saw that, but when run in node.js v16.19.0, the default fields *do* work. If you would like me to create manual defaults in parseCliArguments, I can do so; but in the current node.js v16 version, it works as it is.

Reply:
Didn't test. Good to know. As I said it doesn't matters.